calico_cni/calico_cni.py

```diff
@@ -245,16 +246,17 @@ def delete(self):
 
         _log.info("Finished removing container: %s", self.container_id)
 
-    def _assign_ip(self):
+    def _assign_ip(self, env):
```

Comment:
Do we ever pass anything besides `self.env` to this method?

I'm not sure that an `env` argument is really required for `_assign_ip`, in which case we should just remove it.